.github/workflows/destroy-environment.yml

```diff
@@ -5,17 +5,17 @@ on:
   # Ability to execute on demand
   workflow_dispatch:
     inputs:
-      ec-api-key:
-        type: string
-        description: "Elastic Cloud API key"
-        required: true
       prefix:
         type: string
         description: "Delete all environments starting with `prefix`"
         required: true
       ignore-prefix:
         type: string
         description: "Ignore all environments starting with `ignore-prefix`"
+      ec-api-key:
+        type: string
+        description: "**Optional** To delete env environments on your own organization, enter your Elastic Cloud API key."
+        required: true
```

Comment:
I want to clarify the definition of `ec-api-key` as optional. Despite its `required: true` setting, users must manually input values for the required inputs before the workflow can run, as specified in the [GitHub docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs). If we intend to treat it as optional, it should be interpreted as `required: false`. In that case, we would need to develop logic within an action (job step) to use the default API key defined as a secret when no API key is provided.